composer.json

```diff
@@ -142,16 +141,6 @@
 			"rm -f ./src/generated/container.php.meta",
 			"composer du",
 			"Yoast\\WP\\Free\\Composer\\Actions::compile_dependency_injection_container"
-		],
-		"post-install-cmd": [
-			"xrstf\\Composer52\\Generator::onPostInstallCmd",
-			"Yoast\\WP\\Free\\Composer\\Actions::prefix_dependencies"
```

Comment:
I believe we still need this for our current dependencies (which are needed in the `src` directory).

Reply:
@andizer Oops... :flushed: Thank you for catching that!

Looking at the scripts again, I wonder why that same command wasn't added to the `post-update-cmd` script ? That doesn't seem right.

I've fixed this in a separate commit for easier reviewing. Let me know if you'd like me to squash the commits.